source/style.rst

```diff
@@ -0,0 +1,171 @@
+=======================
+PCDS Python Style Guide
+=======================
+As much as possible, PCDS code should be written with the following guidelines in
+mind.
+
+Code Style
+==========
+Our code follows `PEP-8`_ guidelines. Please look there for any code style matters. Since this is a very widely used standard, Google is also very helpful
```

Comment:
Should break this line up a bit to make the docs source file more manageable

Reply:
Oops, yeah. Wasn't using my regular text editor for this so the text just wrapped and I forgot to fix.